source/develop/release-management/features/virt/enhance-import-export-with-ova.md

```diff
@@ -54,82 +54,83 @@ The following figure depicts the import process of an OVA that was generated by
-2. In the figure above the format of the disks within the OVA is QCOW that is going to be the default format in OVA files generated by oVirt. However, we should support import of OVA that contains RAW disks as well.
-3. In the figure above the destination volumes are of type RAW. While this is preferable because these volumes would serve as base volumes and the performance is better by having them of type RAW, the conversion of volumes on-the-fly from QCOW to RAW may be postponed and so at the beginning the target volumes would be of type QCOW.
+1. The described process assumes that the disks are not compressed within the OVA. In order to support this, the back end would need to pass an indication to imageio-proxy as to whether or not each disk is compressed (step 7), so the proxy can decompress the data on-the-fly while streaming it to the destination volume (step 10).
+2. In the figure above, the format of the disks within the OVA is QCOW. This will serve as the default format in all OVA files generated by oVirt. However, we should also support import of OVA files containing RAW format disks.
+3. In the figure above, the destination volumes are in the RAW format. This is preferable format because these volumes will serve as base volumes, and the RAW format delivers better performance. However, the destination volumes may initially be in the COW format (copied as-is from the OVA file) and only later on converted to the RAW format.  
 
 ### Uploading an OVA Generated by an External Source
-The following figure depicts the import process of an OVA that was generated by external source:
+The following figure depicts the import process of an OVA file that was generated by an external source:
 
 ![](../../../../images/wiki/upload-external-ova.png)  
 
-The process differ from import of OVA generated by oVirt in the following steps:  
+The process is similar to that of importing an OVA file generated by oVirt, except for the following steps:  
 6. Only one temporary disk is created on the storage domain.  
-10. The whole OVA is streamed into the created disk.  
-11. Additional step that converts the streamed OVA and removed the temporary disk.  
+10. The whole OVA file is streamed into the disk that has just been created.  
+11. Convert the temporary disk to a VM using virt-v2v.  
+12. Remove the temporary disk.  
 
 Note:  
 
-1. This process if not efficient in time and space, due to the creation of the temporary disk. Thus, we will suggest to make the OVA accessible to the host instead, especially in the case of mass import of OVA files. However, this process can be handy for a one upload with loose time- and space- constraints.  
+1. This process is not efficient in time and space, due to the creation of the temporary disk. Thus, we will suggest to make the OVA accessible to the host instead, especially in the case of mass import of OVA files. However, this process can be handy for importing one OVA file with loose time and space constraints.  
```

Comment:
@ahadas Suggest to replace "Thus," with "Therefore," (it's more for this context)